app/models/case_contact_report.rb

```diff
@@ -25,6 +25,7 @@ def filtered_case_contacts(args)
       .contact_type(args[:contact_type_ids])
       .contact_type_groups(args[:contact_type_group_ids])
       .with_casa_case(args[:casa_case_ids])
+      .order(:id)
```

Comment:
Could you change the test instead of the app code?

Reply:
Sure no problem. Can I ask why it is preferable to go with Active Record's flaky ordering instead of explicit ordering? Happy to make the change, just pure curiosity.

Reply:
Generally you don't wanna change the app code to make a test run. You want to just change the test.

Reply:
Ah got it, thank you :)